classroom.rb

```diff
@@ -0,0 +1,22 @@
+class Classroom
+  attr_accessor :label, :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    students << student
```

Comment:
- [ ] @Tafloninno great work on your classroom class, however in this line you made a mistake of adding student to `students` instead of `@students` which isthe array name you created in line 6, kindly consider rectifying it.

Reply:
- [ ] @Tafloninno great work adding a method `that serves as the 'has-many' side (a classroom has many students)`, however, you forgot to add a method in your student class `that serves as the 'belongs-to' side (a student belongs to a classroom)` kindly consider adding it.